Mathlib/Order/Defs/PartialOrder.lean

```diff
@@ -53,30 +55,35 @@ lemma le_rfl : a ≤ a := le_refl a
 /-- The relation `≤` on a preorder is transitive. -/
 lemma le_trans : a ≤ b → b ≤ c → a ≤ c := Preorder.le_trans _ _ _
 
-lemma lt_iff_le_not_le : a < b ↔ a ≤ b ∧ ¬b ≤ a := Preorder.lt_iff_le_not_le _ _
+lemma lt_iff_le_not_ge : a < b ↔ a ≤ b ∧ ¬b ≤ a := Preorder.lt_iff_le_not_ge _ _
+
+@[deprecated (since := "2025-05-11")] alias lt_iff_le_not_le := lt_iff_le_not_ge
+
+lemma lt_of_le_not_ge (hab : a ≤ b) (hba : ¬ b ≤ a) : a < b := lt_iff_le_not_ge.2 ⟨hab, hba⟩
 
-lemma lt_of_le_not_le (hab : a ≤ b) (hba : ¬ b ≤ a) : a < b := lt_iff_le_not_le.2 ⟨hab, hba⟩
+@[deprecated (since := "2025-05-11")] alias lt_of_le_not_le := lt_of_le_not_ge
 
 lemma le_of_eq (hab : a = b) : a ≤ b := by rw [hab]
-lemma le_of_lt (hab : a < b) : a ≤ b := (lt_iff_le_not_le.1 hab).1
-lemma not_le_of_lt (hab : a < b) : ¬ b ≤ a := (lt_iff_le_not_le.1 hab).2
-lemma not_le_of_gt (hab : a > b) : ¬a ≤ b := not_le_of_lt hab
-lemma not_lt_of_le (hab : a ≤ b) : ¬ b < a := imp_not_comm.1 not_le_of_lt hab
-lemma not_lt_of_ge (hab : a ≥ b) : ¬a < b := not_lt_of_le hab
+lemma le_of_lt (hab : a < b) : a ≤ b := (lt_iff_le_not_ge.1 hab).1
+lemma not_le_of_gt (hab : a < b) : ¬ b ≤ a := (lt_iff_le_not_ge.1 hab).2
+lemma not_lt_of_ge (hab : a ≤ b) : ¬ b < a := imp_not_comm.1 not_le_of_gt hab
```

Comment:
I would have thought the names would be:
```suggestion
lemma not_ge_of_lt (hab : a < b) : ¬ b ≤ a := (lt_iff_le_not_ge.1 hab).2
lemma not_gt_of_le (hab : a ≤ b) : ¬ b < a := imp_not_comm.1 not_le_of_gt hab
```
since the conclusions have their order swapped relative to the hypotheses, which come first? (And also swapped relative to the order `a`, `b` of the arguments?)

Reply:
I don't feel too strongly as long as there's documented agreement, so if you can get the naming guide PR merged with your version I'm happy too. I'll just say that this particular aspect of the naming convention did not immediately come to mind when I was reading the changes here (though I've spent much less time naming things than you have!)